### ISSUE TITLE
not finding the config file

### ISSUE THREAD

**Issue by 1stchiller** [GreyHak/dsp-drone-clearing#10]
I'm unable to find the config file and my drones aren't clearing. 
The Mecha still runs up to the desginated location and clears by himself. The game was running in main menu and I was searching at the given location for the file (running win10) and the latest version of the game (0.6.17.5831)

I'm running the following mods as well:
BepInEx 5.4.5
CopyInserters 1.6.0
LDBTool 1.8.0
More_Smelters 0.3.1
AssemblerVerticalConstruction 1.0.5
SmerlterMiner 1.3.0
SplitterOverConveyer 2.0.0
AdvancedMiner 1.0.5
MinerInfo 1.2.10
DSP_Belt_Reverse_Direction 1.0.0

**Comment by GreyHak** [comment]
@1stchiller Thank you for the report.  Could you please confirm you're on the latest v.1.2.10 version of this mod.  Could you also please attach the BepInEx log file which you should be able to find in "C:\Program Files (x86)\Steam\steamapps\common\Dyson Sphere Program\BepInEx\LogOutput.log" or your equivalent folder.

**Comment by 1stchiller** [comment]
thanks for the reply.
I'm working on the latest version. 
Drones flying out now... Rebooted my pc and the game placed some solar panels (i had inventory space before).

There must be something wrong with my game though. i have no such file.

**Comment by GreyHak** [comment]
@1stchiller In your reply "Drones flying out now", I'm guessing you're referring to the placing of solar panels, and that clearing is still not working.

If you have BepInEx installed (as you've indicated you do), and you don't have a `LogOutput.log` file, then it's probably disabled.  It's enabled by default, but you may have changed it.  In `BepInEx\config\BepInEx.cfg` there's a setting under [Logging.Disk] called Enable which should be set to true.  Note that there are two Enable settings, so make sure to set the one in the [Logging.Disk] section.  If you're unsure which one, just set them both to True.  Then run the game, and you should get a `LogOutput.log` file in your `Dyson Sphere Program\BepInEx` folder.

To debug the issue it might also be helpful for you to disable the other mods.  If that fixes the problem, then we know we're dealing with a mod conflict.  If that's the case, I'll need your help to narrow down which mod is causing the conflict so I can try to fix it.

**Comment by 1stchiller** [comment]
@GreyHak hey thanks for your effort. I meant that after I cleared some more inventory slots, like half of it, the drones flew out and gathered trees and stones. So that issue was resolved. Although I haven't figured out yet what requirements need to be met, so the drones go out and about. The button in HUD was always activated.

The other thing (main issue) is that I can't find the config file. I've installed all mods with the r2modman and at the given location the only file I have is from the planetary-copy-mod (it's a blueprint file) which I have deactivated due to not comprehending the mod.

I have no logfile from BepInEx and haven't disabled anything. It just worked for me without changing stuff in the config files. 
So it might not even be an issue with your mod but more with me 😅.

**Comment by 1stchiller** [comment]
@GreyHak Thank you for your great help!
I found the file and can explain many things now. 
1. the drones weren't flying out because i had less than 10 free inventory slots
2. not enough mecha power

keep up the good work. My main Issue of not finding the config file was resolved and every other issue was about not knowing what requirements (min. free inventory spaces, min. energy) needed to be met.
